### PR TITLE
MOSIP-20444 MOSIP-19990 registration-client-sdk-integration.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by pjoshi751 — COMMENTED** [review]
1. Name consistency - we are calling it "Biometric SDK".  Lets use the name consistently.  Please match it with biometric-sdk.md.  You may change it to a better one.
2. Doc conventions - headers,  code in code-like text etc.
3. Link this doc to biometric-sdk.md.  Let the description of what  SDK does go in biometric-sdk.md.  This is pure integration guide.  It may cross reference parent page.

File: docs/registration-client-home-page.md [hunk 1]
```diff
@@ -112,9 +112,19 @@ When the operator clicks on the **Capture** button and tries to capture the biom
...
- What is the difference between an adult' and an infant' biometric capture?
+What is the difference between an adult' and an infant' biometric capture?
  * For an adult, all the configured biometrics can be captured.
- * For an infant, by default, only the face biometrics is allowed to be captured.   
+ * For an infant, by default, only the face biometrics is allowed to be captured.
+    
+    
+4. **Concept of biometric exception**
+* Permanent or temporary missing / defective fingers or irises can be marked as exception during registration process.
+* Marked exception finger / iris names are sent as part of rcapture request to SBI.
```
Comment (SwatiPeriwal) [inline docs/registration-client-home-page.md]: `rcapture`

File: docs/registration-client-home-page.md [hunk 2]
```diff
@@ -112,9 +112,19 @@ When the operator clicks on the **Capture** button and tries to capture the biom
...
+* A photo of resident is captured highlighting his/her biometric exceptions called as Proof of exception (POE).
+* Biometric exception photo is captured by the biometric face camera device.
+* Until 1.2.0, POE was collected only as document type field.
```
Comment (SwatiPeriwal) [inline docs/registration-client-home-page.md]: **document type** --  its a specfic field name

File: docs/registration-client.md [hunk 3]
```diff
@@ -18,26 +20,32 @@ The Registration client can be operated by an operator who can be either a **Sup
...
 ## Data protection
 
-## Packets
+* We store registration packets and synced data in the client machine.
+* Most of the synced data are stored in the derby DB. Derby DB is encrypted with the bootpassword.
+* Derby DB boot password is encrypted with machine TPM key and stored under `.mosipkeys/db.conf`.
+* Synced UI-SPEC/script files are saved in plain text under registration client working directory. During sync, SPEC/script file hash is stored in derby and then the files are saved in the current working directory. Everytime the file is accessed by the client performs the hash check.
+* Synced pre-registration packets are encrypted with TPM key and stored under configured directory.
+* Directory to store the registration packets and related registration acknowledgments is configurable. 
+* Regiatration packet is an signed and encrypted ZIP.
```
Comment (SwatiPeriwal) [inline docs/registration-client.md]: Registration spell check.

File: docs/registration-client-home-page.md (hunk 2 above)
Comment (ghost) [reply docs/registration-client-home-page.md]: done

File: docs/registration-client.md (hunk 3 above)
Comment (ghost) [reply docs/registration-client.md]: Done

**Comment by SwatiPeriwal** [comment]
Reviewed.. can be merged..

**Pull request merged**